modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro

```diff
@@ -0,0 +1,18 @@
+definition {
+
+	macro metricsWorkflowSearchEmpty {
+		var key_workflowSearchName = "${workflowSearchName}";
+
+		Type(
+			key_fieldLabel = "Search for",
+			locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR",
+			value1 = "${workflowSearchName}");
+
+		Click(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR_BUTTON");
+
+		AssertElementNotPresent(
+			key_workflowProcessName = "${workflowProcessName}",
+			locator1 = "WorkflowMetrics#METRICS_WORKFLOW_PROCESS_NAME");
```

Comment:
Please, remove this, we need to have 2 different macros: 1 for the search (so it can be used for other tests) and 1 for to assert.

Also, this assert it not correct for this test. The purpose of this test is to verify that the text "No results were found." is present when searching for a term that does not exist. Then, create another macro, something like `assertSearchEmptyState` that will verify that this text is present.